qpdf/qtest/qpdf/test-90.out

```diff
@@ -1,5 +1,5 @@
 WARNING: various-updates.json, trailer at offset 606: operation for array attempted on object of type dictionary: ignoring attempt to append item
 WARNING: various-updates.json, obj:7 0 R at offset 197: operation for array attempted on object of type dictionary: ignoring attempt to append item
-WARNING: various-updates.json, obj:7 0 R at offset 309: operation for integer attempted on object of type array: returning 0
+WARNING: various-updates.json, obj:0 0 R at offset 309: operation for integer attempted on object of type array: returning 0
```

Comment:
and here?

Reply:
The current commits reproduce the existing output.